Dockerfile

```diff
@@ -12,24 +12,25 @@ WORKDIR /app
 
 COPY poetry.toml poetry.lock pyproject.toml ./
 
-RUN poetry install --no-dev
+RUN poetry config virtualenvs.create false --local &&\
+    poetry install
 
 COPY . .
 
 FROM base as prod
-ENTRYPOINT poetry run gunicorn --bind 0.0.0.0:5000 "app:create_app(None)"
+CMD poetry run gunicorn --bind 0.0.0.0:$PORT "app:create_app(None)"
```

Comment:
I mentioned having a default port in my other comment. You could do that here with `${PORT:-5000}`